### PR TITLE
Update to 1.19

### PR DESCRIPTION
**Pull request by fedetrifo06**
I updated the mod to 1.19
The mod work and i don't found any problems

### CODE REVIEW

**Comment by SrIzan10** [comment]
how to compile it?

**Comment by piebaker** [comment]
1.19.1? Building on my end leads to fabric yelling about incorrect versioning even with all variables changed. Can you share your final. jar file?

**Comment by Grawl** [comment]
Yes please let us test it without compiling

**Comment by kirmerk1** [comment]
> I updated the mod to 1.19 The mod work and i don't found any problems

Can't build it. Building gives me an error. Please upload JAR to Google Drive or something.

File: src/main/resources/seamless_loading_screen.mixins.json [hunk 1]
```diff
@@ -10,7 +10,6 @@
     "GameRendererMixin",
     "LevelLoadingScreenMixin",
     "LevelStorageMixin",
-    "MinecraftClientMixin",
```
Comment (magicus) [inline src/main/resources/seamless_loading_screen.mixins.json:13]: This does not seem correct.. Is not the mixin is still needed? And if it is not, why not remove it?

**Comment by Dragon-Seeker** [comment]
1.19 version will be out within the coming days, thank you for the PR.

**Pull request closed without merging**